apps/webhook/tests/unit/test_service_pool.py

```
@@ -0,0 +1,101 @@
+@pytest.mark.asyncio
+async def test_service_pool_close():
+    """Test that ServicePool can be closed."""
+    pool = ServicePool.get_instance()
+
+    # Should not raise
+    await pool.close()
+
+
+def test_service_pool_reset():
+    """Test that ServicePool can be reset (for testing)."""
+    pool1 = ServicePool.get_instance()
```

Ruff (0.14.4)

Local variable `pool1` is assigned to but never used

Remove assignment to unused variable `pool1`

(F841)